firmware/linker/stm32h733vgtx/stm32h733vgtx_app.ld

```diff
@@ -103,7 +103,8 @@ SECTIONS
     *(.bss)
     *(.bss*)
     *(COMMON)
-
+    KEEP(*(.bootloader_byte))
```

Comment:
what does this do here? Is this the bootloader bit we were talking about earlier?

Reply:
Yes, it's from the bootloader bit. Forgot to remove it